lib/spree/models/line_item_decorator.rb

```diff
@@ -0,0 +1,11 @@
+module Spree
+  module LineItemDecorator
+    protected
+
+    def unit_price
+      price
+    end
+
+    ::Spree::LineItem.prepend self
+  end
+end
```

Comment:
Obviously just a tiny nit pick, but if you leave newlines at the end of files then GitHub won't leave these icons saying they're missing. I have my editor configured to do this automatically.